nginx/ngx_http_js_module.c

```diff
@@ -76,6 +77,11 @@ typedef struct {
     uintptr_t              handler;
 }  ngx_http_js_header_t;
 
+typedef struct {
+    ngx_str_t fname;
+    unsigned flags;
+}  ngx_http_js_set_t;
```

Comment:
We have two modules: HTTP and stream. For symmetry this feature should be added to both modules.

Therefore put constant and `ngx_http_js_set_t` named as `ngx_js_set_t` to `nginx/ngx_js.h` just under `ngx_js_named_path_t` typefef.